README.md

```diff
@@ -1,13 +1,5 @@
 # Express API Project
+This project's goal is to create my first RESTful API using Node in Express. This API should have a series of endpoints which return an array of data or a single object. I worked with a hardcoded dataset in JSON format with a list of 500 books and their details. I created the different endpoints by manipulating the data with JavaScript methods like: filter(), find() and slice().
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+## View it live - almost alive
+https://express-api-project-horvath.herokuapp.com/ 
```

Comment:
It would be nice to provide a list of endpoints with descriptions of what data they are providing 😄